### PR TITLE
Remove functionality

### PR DESCRIPTION
**Pull request by dcoleman21**
Removed from the Welcome page and Learn More page:
- [x] track your carbon...
- [x] connect with community...

Added:
- [x] w3-center to the shared/_login_with_google file to re-center the title on the Learn More page

### CODE REVIEW

File: app/views/shared/_login_with_google_button.html.erb [hunk 1]
```diff
@@ -3,13 +3,11 @@
   <%= link_to "Login with Google", "/auth/google_oauth2", class: "w3-btn w3-blue-grey w3-block" %>
 </div>
 
-<div class="w3-panel w3-pale-blue w3-block">
+<div class="w3-panel w3-pale-blue w3-block w3-center">
```
Comment (LHJE) [inline app/views/shared/_login_with_google_button.html.erb:6]: Looks great!

**Review by LHJE — APPROVED** [review]
Nice!

File: spec/features/welcome/index_spec.rb [hunk 2]
```diff
@@ -49,8 +49,6 @@
       expect(page).to have_content("Login with Google to:")
       expect(page).to have_content("- Set up a garden -")
       expect(page).to have_content("- Track your sensor data -")
-      expect(page).to have_content("- Connect with your community -")
-      expect(page).to have_content("- Track your garden's carbon impact -")
```
Comment (lkriffell) [inline spec/features/welcome/index_spec.rb:53]: Sad we couldn't implement this, it was a great idea!

**Review by lkriffell — APPROVED** [review]
Good stuff. Looks like there is a merge conflict for a yml file. I don't know the FE code base that well otherwise I would say get rid of it lol.

File: spec/features/welcome/index_spec.rb (hunk 2 above)
Comment (leahriffell) [reply spec/features/welcome/index_spec.rb:53]: Agreed but I have faith that we can someday!

**Comment by leahriffell** [comment]
Agreed with Logan I think it's fine to just delete the yml file you have that has conflicts! that test should be switched to a webmock anyway

**Pull request merged**